dataset/dictionary/enemies.ts

```diff
@@ -3001,6 +3001,20 @@ export default [
     zhTW: "爐殼山鼬",
     tags: [ "natlan", "enemy", "living-being" ],
   },
+  {
+    en: "Capybara Great Majesty!",
+    ja: "カピバラ キング!",
+    zhCN: "豚兽 大王!",
+    zhTW: "豚獸 大王!",
+    tags: [ "natlan", "enemy", "living-being" ],
+  },
+  {
+    en: "Bakunawa",  // note: It may become an enemy-boss in the future. You can rematch it every two days after the battle.
+    ja: "バクナワ",
+    zhCN: "巴窟纳瓦",
+    zhTW: "巴窟納瓦",
+    tags: [ "natlan", "enemy" ],
```

Comment:
You can tag it with `enemy-boss` now. I already added event bosses such as Osial (オセル).

```suggestion
    en: "Bakunawa",
    ja: "バクナワ",
    zhCN: "巴窟纳瓦",
    zhTW: "巴窟納瓦",
    tags: [ "natlan", "enemy-boss" ],
```